### PR TITLE
Enable feature tests

### PR DESCRIPTION
**Pull request by sullivangareth**
### Change description ###

Re-enable feature tests that had previously been disabled (see commit below) due to IDAM issues in AAT environment

https://github.com/hmcts/aac-manage-case-assignment/commit/10eb2aa912c3a28ddb9af7df8d44590de6b2f07d

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -288,7 +288,7 @@ dependencies {
   integrationTestImplementation sourceSets.main.runtimeClasspath
   integrationTestImplementation sourceSets.test.runtimeClasspath
 
-  testImplementation group: 'uk.gov.hmcts', name: 'befta-fw', version: '5.1.0'
+  testImplementation group: 'uk.gov.hmcts', name: 'befta-fw', version: '5.16.1'
```
Comment (smathangi) [inline build.gradle:291]: I don't think current version of definition-store on AAT supports NoC elements in the spreadsheet.

**Review by smathangi — COMMENTED** [review]
Thank you for raising this PR. I think we should open a high priority ticket when they were disabled.

**Review by MSancaktutar — APPROVED** [review]
LGTM

File: build.gradle (hunk 1 above)
Comment (MSancaktutar) [reply build.gradle:291]: We are using def store's develop branch, aren't we?

Comment (sullivangareth) [reply build.gradle]: reverted to 5.1.0

Comment (MSancaktutar) [reply build.gradle]: Sorry - you were right, @smathangi . Thanks for reverting @sullivangareth .

**Pull request merged**